### DISCUSSION TITLE
Error in reply_communication_showcase.Rmd

### DISCUSSION THREAD

**Discussion by anthonyjlau** [sailuh/kaiaulu#274] [Q&A] [answered]
I get an error in reply_communication_showcase.Rmd when trying to create the project_log variable: 
https://github.com/sailuh/kaiaulu/blob/9294a9bb92506b76424b0fcc7c07f6b74d5ce53d/vignettes/reply_communication_showcase.Rmd#L54-L58

The error is: "Error: Elements listed in 'by.x' must be valid column names in x." The by.x parameter is equal to name_column[i] and name_column is a vector that has 'reply_from' twice. 

I also get other errors on lines 92, 131, and 137 but I suspect these errors stem from the first error. 
https://github.com/sailuh/kaiaulu/blob/9294a9bb92506b76424b0fcc7c07f6b74d5ce53d/vignettes/reply_communication_showcase.Rmd#L92
https://github.com/sailuh/kaiaulu/blob/9294a9bb92506b76424b0fcc7c07f6b74d5ce53d/vignettes/reply_communication_showcase.Rmd#L131
https://github.com/sailuh/kaiaulu/blob/9294a9bb92506b76424b0fcc7c07f6b74d5ce53d/vignettes/reply_communication_showcase.Rmd#L137

**Comment by carlosparadis** [comment]
As we said on the call, you must download a mailing list data to use it. This means the notebook documentation does not explain what this Notebook is trying to do. Please let me know if you cannot download the mailing list if the notebook does not go over that, as it should also be added to the narrative.

**Reply by anthonyjlau** [reply, marked as answer]
Hi, I was able to find and download the mailing list data and this solved my errors. The .mbox file that I needed to download was from a link in the geronimo.yml file. I went to that link, searched for "geronimo", and clicked the download button. Then I changed the name of the .mbox file to match the file name in the mbox part of the geronimo.yml file.

As a side note, on the [Third Party Tools Setup](https://github.com/sailuh/kaiaulu/wiki/Third-Party-Tools-Setup) page, it says to run _vignettes/mailinglist_showcase.Rmd_ but that vignette does not exist.
